staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -618,16 +619,16 @@ See https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts`
 		if err != nil {
 			return err
 		}
-		patchBytes, patchedObject, err := patcher.Patch(info.Object, modified, info.Source, info.Namespace, info.Name, o.ErrOut)
+		patchedObject, err := patcher.Patch(info.Object, modified, info.Source, info.Namespace, info.Name, o.ErrOut)
 		if err != nil {
-			return cmdutil.AddSourceToErr(fmt.Sprintf("applying patch:\n%s\nto:\n%v\nfor:", patchBytes, info), info.Source, err)
+			return cmdutil.AddSourceToErr(fmt.Sprintf("applying patch:\n%v\nto:\n%v\nfor:", patchedObject, info), info.Source, err)
```

Comment:
Now that we are logging the entire patched object instead of a patch string, I wonder if this log has changed significantly.

Reply:
I agree with this comment from Alex, this change isn't too helpful, since you're basically printing the resource after the patch was applied, so either we'd need to change the message to say so explicitly, or double check if the error returned from patcher does not contain the patch contents itself. Maybe embeding the patch there would also make sense and would help us to not to have to expose it here directly, but rather through that error.

Reply:
We are already calling `AddSourceToErr` in here https://github.com/kubernetes/kubernetes/blob/1d2f3089ce7f22282b77adc2b7e5856249c0142a/staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go#L247. What if we directly return err instead printing the patch diff?